src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefLocalTests.cs

```diff
@@ -3069,6 +3069,243 @@ .maxstack  4
 }");
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/71369")]
+        public void AssignmentValue_RefField()
```

Comment:
Note: only the first two tests (`AssignmentValue_RefField` and `AssignmentValue_RefField_Temp`) are directly for the bug, the others capture existing behavior which hasn't changed.